vignettes/articles/KRI Method.Rmd

```diff
@@ -17,7 +17,8 @@ knitr::opts_chunk$set(
 
 # Overview
 
-This vignette outlines the statistical methods used to evaluate Key Risk Indicators (KRIs) in {gsm}. KRIs are metrics that allow users to measure pre-defined risks and determine the level of observed risk to data quality and patient safety in a clinical trial. The {gsm} package implements a standardized data pipeline to facilitate KRI analysis. Other vignettes provide an overview of this framework ([1](https://gilead-biostats.github.io/gsm/), [2](https://gilead-biostats.github.io/gsm/articles/Cookbook.html) [3](https://gilead-biostats.github.io/gsm/articles/DataPipeline.html), [4](https://gilead-biostats.github.io/gsm/articles/DataAnalysis.html)), and the statistical methods for this process are described in detail below.
+This vignette outlines the statistical methods used to evaluate Key Risk Indicators (KRIs) in {gsm}. KRIs are metrics that allow users to measure pre-defined risks and determine the level of observed risk to data quality and patient safety in a clinical trial. The {gsm} package implements a standardized data pipeline to facilitate KRI analysis. Other vignettes provide an overview of this framework ([1](https://gilead-biostats.github.io/gsm/), [2](https://gilead-biostats.github.io/gsm/articles/Cookbook.html) [3](https://gilead-biostats.github.io/gsm/articles/DataModel.html), [4](https://gilead-biostats.github.io/gsm/articles/DataAnalysis.html),
+[5](https://gilead-biostats.github.io/gsm/articles/DataReporting.html)), and the statistical methods for this process are described in detail below.
```

Comment:
Hmmm. Formatting-wise this is probably best to keep as-is. If the site moves or something (eg, we put it under the OpenRBQM umbrella) this will be easy to miss, but let's just make sure we do a search if that happens. Unless we want to spell these out in long-form.

Reply:
Oh duh this one is online only, so it's fine. But... we shouldn't need the full paths, so I'm suggesting that change to make this more portable.
```suggestion
This vignette outlines the statistical methods used to evaluate Key Risk Indicators (KRIs) in {gsm}. KRIs are metrics that allow users to measure pre-defined risks and determine the level of observed risk to data quality and patient safety in a clinical trial. The {gsm} package implements a standardized data pipeline to facilitate KRI analysis. Other vignettes provide an overview of this framework ([1](../), [2](Cookbook.html) [3](DataModel.html), [4](DataAnalysis.html),
[5](DataReporting.html)), and the statistical methods for this process are described in detail below.
```